packages/next/build/polyfills/fetch.js

```diff
@@ -0,0 +1 @@
+export default window.fetch
```

Comment:
Why is this `export default` and `Object.assign` using `module.exports`? If they're intentionally different, I'd prefer to see a comment explaining each!